torch/testing/_internal/common_methods_invocations.py

```diff
@@ -17237,14 +17237,12 @@ def _compare_trilu_indices(
             torch.triu_indices(row, col, offset, dtype=dtype, device=device))
 
     else:
-        # TODO(#38095): Replace assertEqualIgnoreType. See issue #38095
-        self.assertEqualIgnoreType(
+        self.assertEqual(
```

Comment:
Simple renaming will fail. Some code changes are needed to align the types.

Reply:
Since there was an explicit conversion of both tensors to dtype, I though the dtypes would be consistent.

Also Just updated with a new commit: Added "device" to the "to" function to make sure that the data-types are aligned.

Let me know if I am missing something here.